.eslintrc

```diff
@@ -71,6 +71,7 @@
     "strict": 0,
     "valid-jsdoc": 2,
     "valid-typeof": 2,
-    "wrap-iife": [ 2, "inside" ]
+    "wrap-iife": [ 2, "inside" ],
+    "object-shorthand": [ 2, "methods" ]
```

Comment:
Sure. Fair enough. It was me who didn't notice that they are sorted alphabetically.